src/frontend/screens/Settings/ProjectSettings/YourTeam/ReviewAndInvite/index.tsx

```diff
@@ -19,41 +21,56 @@ export const ReviewAndInvite: NativeNavigationComponent<'ReviewAndInvite'> = ({
   route,
   navigation,
 }) => {
-  const [inviteStatus, setInviteStatus] = React.useState<
-    'reviewing' | 'waiting'
-  >('reviewing');
   const {role, deviceId, deviceType, name} = route.params;
 
   const {openSheet, sheetRef, closeSheet, isOpen} = useBottomSheetModal({
     openOnMount: false,
   });
-  const project = useProject();
-  const queryClient = useQueryClient();
+  const sendInviteMutation = useSendInvite();
+  const requestCancelInviteMutation = useRequestCancelInvite();
 
   function sendInvite() {
-    setInviteStatus('waiting');
-    project.$member
-      .invite(deviceId, {roleId: role})
-      .then(val => {
-        if (val === 'ACCEPT') {
-          queryClient.invalidateQueries({queryKey: ['projectMembers']});
-          navigation.navigate('InviteAccepted', route.params);
-          return;
-        }
+    sendInviteMutation.mutate(
+      // @ts-ignore
```

Comment:
Is this error happening because `role` is invalid? If so, this was fixed in 1a34eb3f5055fc010f968ca2f0264eb0a2927030. You should be able to merge `main` to pick this up.